frigate/object_processing.py

```diff
@@ -659,9 +659,14 @@ def on(self, event_type: str, callback: Callable[[dict], None]):
     def update(self, frame_time, current_detections, motion_boxes, regions):
         # get the new frame
         frame_id = f"{self.name}{frame_time}"
-        current_frame = self.frame_manager.get(
-            frame_id, self.camera_config.frame_shape_yuv
-        )
+
+        try:
```

Comment:
Should we put this try/except in the frame manager itself and return `None` if the frame isn't found?